api/src/services/settings.js

```diff
@@ -49,6 +49,17 @@ const getDeprecatedTransitions = () => {
     }));
 };
 
+const hasOidcProvider = async () => {
+  const settingsDoc = await getDoc();
+  if(!settingsDoc){
+    return false;
+  }
+  if(!!settingsDoc.settings && !!settingsDoc.settings.oidc_provider){
+    return true;
+  }
+  return false;
+};
```

Comment:
suggestion: we should be able to simplify this a bit.  The server will actually fail to start without a settings doc, so we do not need to explicitly handle that case here.  

```suggestion
const hasOidcProvider = async () => {
  const settings = await module.exports.get();
  return !!settings?.oidc_provider;
};
```